ouroboros-consensus/test/consensus-test/Test/Consensus/HardFork/History.hs

```diff
@@ -503,7 +519,13 @@ data EventTime = EventTime
   { eventTimeSlot :: SlotNo
   , eventTimeEpochNo :: EpochNo
   , eventTimeEpochSlot :: Word64
+  -- ^ Relative slot withing the current epoch round,
+  --   needed to be able to advance the epoch number
   , eventTimeRelative :: RelativeTime
+  , eventTimePerasRoundNo :: PerasRoundNo
+  , eventTimeSlotInPerasRound :: Word64
+  -- ^ Relative slot withing the current Peras round,
```

Comment:
```suggestion
  -- ^ Relative slot within the current Peras round,
```

Reply:
Fixed in f0e7a56d23e4a5da846b820e3edf9497d3887fdd